src/amp-story-player/amp-story-player-impl.js

```diff
@@ -1048,6 +1075,38 @@ export class AmpStoryPlayer {
     return this.iframes_[this.stories_[nextStoryIdx][IFRAME_IDX]];
   }
 
+  /**
+   * Checks if index is out of bounds
+   * @private
+   * @param {number} index
+   * @return {boolean}
+   */
+  isIndexOutofBounds_(index) {
+    return (
+      this.currentIdx_ + index >= this.stories_.length ||
+      this.currentIdx_ + index < 0
+    );
+  }
+
+  /**
+   * Checks if circular wrapping is set
+   * @private
+   * @return {boolean}
+   */
+  circularWrapping_() {
+    return this.element_.getAttribute('enable-circular-wrapping') === 'true';
```

Comment:
Let's just check for the presence of the attribute, so that publishers don't have to specify the value:

`<amp-story-player enable-circular-wrapping>`

then just change this to `this.element.hasAttribute('enable-circular-wrapping')`